README.md

```diff
@@ -90,8 +90,14 @@ Get the tests in `spec/insert_spec.rb` to pass. Input the following 8 bears (you
 
 Get the tests in `spec/select_spec.rb` to pass. Note that for this section, the database will be seeded with external data from the `lib/seed.sql` file so don't expect it to reflect the data you added above. Note: Since it's a Ruby file, write your queries as strings in the `lib/sql_queries.rb`.
 
+You may be expected to use SQL statements that you're not particularly familiar with. Make sure you use the resources and Google to find the right statements. 
+
 ## Resources
 
 [SQL Datatypes](http://www.w3schools.com/sql/sql_datatypes_general.asp)
+[SQL GROUP BY](http://www.w3schools.com/sql/sql_groupby.asp)
```

Comment:
I think we might want to avoid w3schools if we can. Here's SQLite docs for `GROUP BY` https://www.sqlite.org/lang_select.html#resultset

And for data types: https://www.sqlite.org/datatype3.html

:)